index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+        <title>Edwin | Portfolio</title>
+    </head>
+
+    <body>
+        <header>
+            <section class="toolbar">
+                <span class="logo-container">
+                    <h3 class="logo-txt">Edwin G.</h3>
+                </span>
+                <span class="menu-btn">
+                    <img src="./imgs/Union.svg" alt="menu icon">
+                </span>
```

Comment:
- [ ] This is a navigation menu and should be wrapped with the `nav` tag for better semantics. Kindly change that.

Reply:
Done